CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-authn-k8s-client#471](https://github.com/cyberark/conjur-authn-k8s-client/pull/471)
 
 ## [0.24.4] - 2022-05-31
+### Changed
```

Comment:
Is this the correct version? Should it be 0.24.4 or 0.23.4?

Reply:
Should be 0.23.4. Thanks for catching that.

Reply:
Rather than complicate things, I put my change 0.23.5 to match yours.